### PR TITLE
feat(aws-lambda): Add Serverless Framework code snippet for Python

### PR DESCRIPTION
**Pull request by connorads**
Updating docs for two reasons:

1. Want to double check that this is the recommended/correct way to do it 👀
2. Hopefully this example helps other people using Python & `serverless`

### Context
 
Just tried to use `serverless` with Python and the AWS Lambda Layer Integration.

Got something working, with a bit of trial and error, by combining the docs from:
- https://docs.sentry.io/product/integrations/aws-lambda/#serverless-framework
- https://docs.sentry.io/product/integrations/aws-lambda/how-it-works/#python

### Preview

https://sentry-docs-git-fork-connorads-patch-1.sentry.dev/product/integrations/aws-lambda/#serverless-framework

image

### CODE REVIEW

**Comment by HazAT** [comment]
This LGTM but maybe @PeloWriter or @imatwawana could help to make this an include as we have everywhere else.

**Review by ahmedetefy — APPROVED** [review]

File: src/docs/product/integrations/aws-lambda/index.mdx [hunk 1]
```diff
@@ -94,22 +94,53 @@ Learn more about [how this integration works](/product/integrations/aws-lambda/h
...
+provider:
+
+  # other configuration
+  region: <AWS_REGION>
+
+  environment:
+    SENTRY_TRACES_SAMPLE_RATE: '1.0'
+    SENTRY_DSN: '<SENTRY_DSN>'
+
+layersResource:
```
Comment (connorads) [inline src/docs/product/integrations/aws-lambda/index.mdx:133]: What is this `layersResource` thing?

Just tried it with and it didn't work. I might be missing a trick here, perhaps others will too.
```
Serverless: Packaging service...
Serverless: Excluding development dependencies...

 Type Error ----------------------------------------------
 
  TypeError: functionProperties.Layers.forEach is not a function
      at extractLayerConfigurationsFromFunction

  Your Environment Information ---------------------------
     Operating System:          darwin
     Node Version:              16.4.0
     Framework Version:         2.52.0
     Plugin Version:            5.4.3
     SDK Version:               4.2.5
     Components Version:        3.13.4
```

The previous docs suggestion of `custom` worked for me.

File: src/docs/product/integrations/aws-lambda/index.mdx [hunk 2]
```diff
@@ -94,22 +94,53 @@ Learn more about [how this integration works](/product/integrations/aws-lambda/h
...
+  layers:
+    - arn:aws:lambda:${self:provider.region}:943013980633:layer:SentryPythonServerlessSDK:6
+#    - arn:aws:lambda:${self:provider.region}:943013980633:layer:SentryPythonServerlessSDK:latest
+
+<yourFunctionName>:
```
Comment (connorads) [inline src/docs/product/integrations/aws-lambda/index.mdx:138]: Should we stick the function declaration under a [`functions:`](https://www.serverless.com/framework/docs/providers/aws/guide/functions/) or indent it one level?

File: src/docs/product/integrations/aws-lambda/index.mdx (hunk 1 above)
Comment (ahmedetefy) [reply src/docs/product/integrations/aws-lambda/index.mdx]: Thanks for pointing that out!

File: src/docs/product/integrations/aws-lambda/index.mdx (hunk 2 above)
Comment (ahmedetefy) [reply src/docs/product/integrations/aws-lambda/index.mdx]: Nah we should stick with a declaration under `functions:` .. Updated it the snippet :)

**Comment by connorads** [comment]
Just resolved the merge conflicts 💃

Thanks @ahmedetefy for the changes and approval.
What else do we need to do to get this merged?

https://sentry-docs-git-fork-connorads-patch-1.sentry.dev/product/integrations/aws-lambda/#serverless-framework

**Comment by PeloWriter** [comment]
Nothing further, thank you, @connorads. I'm tracking the suggestion to create `include` statements in this [issue](https://github.com/getsentry/sentry-docs/issues/3904)

**Review by PeloWriter — APPROVED** [review]

**Comment by connorads** [comment]
I've now opened up a PR for some examples: https://github.com/getsentry/examples/pull/101

**Pull request merged**